core/app/assets/js/ui/card/card.js

```diff
@@ -0,0 +1,19 @@
+import './card.scss';
+
+export default function Card( props ) {
+	return (
+		<article id={ `type-${ props.type }` } className={ `eps-card ${ props.className }` }>
```

Comment:
Add a prefix to the id

Reply:
But the context is not derermeind at this point it's a generic element...